src/pages/Dashboard/Employees/index.tsx

```diff
@@ -34,8 +62,8 @@ const Box = (): JSX.Element => (
 const EmployeeDashboard = (): JSX.Element => {
   const { employee } = useAuth();
   const { businessId } = useParams();
-  const [employees, setEmployees] = useState([]);
-  const [isOpen, setIsOpen] = useState(false);
+  const [employees, setEmployees] = useState<TableEmployee[]>(initialEmployee);
+  const [formState, setFormState] = useState(FormState.Table);
```

Comment:
This is not descriptive of the situation. When choosing a name, you should be more descriptive of the situation to help other quickly understand what you are doing. Using `formState` could lead someone to think that you are setting the state of a form. Here I would say that this is more like a section.

Reply:
The idea here we have Add Form and Delete From and the employee Table 
By default  the employee table are presented. When clicking on the add button ,  the add form will be shown . Similarity, if clicking on the Delete Button , the Delete from will be displayed. This is the reason I named it  FormState where the states are Table , Add , and DELETE   

if you find this confusing ,what name would you suggest ?

Reply:
This is not resolved. FormState means the state of the form. Like the value in the fields of the form. That is how people will understand this value. Someone will expect the value of the form:
```
{ 
    field1: 'value1',
    field2: 'value2',
    ...
}
```

 but here it's more about the **SECTION** of the dashboard page. This is the section of the employee table, this is the section of the delete form, this is the section of the add employee form. You will get lost when you come later wondering why you chose this name. People that don't know anything about the app and reads your code will wonder what that is and spend time searching the meaning through the code. 

You should be able to find a good name for your variables. It's not up to me to tell you what to choose. Make sure that your reader can understand what you are doing.

Reply:
so , 'whichSectionIsOpen' might be an alternative name . sounds good to you ?